isovar/locus_reads.py

```diff
@@ -29,17 +28,34 @@
 )
 from .common import list_to_string
 from .dataframe_builder import DataFrameBuilder
-
+from .value_object import ValueObject
 
 logger = logging.getLogger(__name__)
 
-class LocusRead(namedtuple("LocusRead", [
+class LocusRead(ValueObject):
+    __slots__ = [
         "name",
         "sequence",
         "reference_positions",
         "quality_scores",
         "base0_read_position_before_variant",
-        "base0_read_position_after_variant"])):
+        "base0_read_position_after_variant"
+    ]
+
+    def __init__(
```

Comment:
> **Since we're aiming for eventual verbosity**: maybe leave the __init__ functions as is?

The bold part is the part I take issue with; I struggle to see the benefit of this. This is the barest abstraction (not the towering monolith of them described in that article—which was really good, by the way) over setting fields in a class. Anyway, not going to insist this change or harp on it any longer, just stating my preference a final time :)